fastlane/lib/fastlane/actions/register_devices.rb

```diff
@@ -184,12 +191,18 @@ def self.example_code
               "Luka iPhone 6" => "1234567890123456789012345678901234567890",
               "Felix iPad Air 2" => "abcdefghijklmnopqrstvuwxyzabcdefghijklmn"
             }
-          ) # Simply provide a list of devices as a Hash',
+          ) # Simply provide a list of devices as a Hash, with the name as key',
```

Comment:
[self-review] Explicitly added comment that name is key in case of using `devices`